java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -753,6 +771,18 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
             1L, // xtrim(streamKey1, new MinId(true, "0-2"))
             OK, // xgroupCreate(streamKey1, groupName1, "0-0")
             OK, // xgroupCreate(streamKey1, groupName1, "0-0", options)
+            true, // xgroupCreateConsumer(streamKey1, groupName1, consumer1)
+            Map.of(
```

Comment:
nit: you can store this as a variable if it ease reading/debugging

Reply:
might be a good idea... ;)